app/javascript/lib/shared/modules/createScaleColors.js

```diff
@@ -1,7 +1,7 @@
 import { scaleOrdinal } from 'd3-scale';
 
 export function createScaleColors(values, arrayDomain) {
-  let colorsGobiertoExtend = ["#12365b", "#118e9c", "#ff766c", "#f7b200", "#158a2c", "#94d2cf", "#3a78c3", "#15dec5", "#6a7f2f", "#55f17b"]
+  let colorsGobiertoExtend = ["#118e9c", "#12365b", "#ff766c", "#f7b200", "#158a2c", "#94d2cf", "#3a78c3", "#15dec5", "#6a7f2f", "#55f17b"]
```

Comment:
Shouldn't we be using here CSS variables of the brand colors? I guess this are already defined.

Reply:
Yes. I've added the Gobierto colors to the `css-confs` as CSS vars.